tockloader-lib/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright OXIDOS AUTOMOTIVE 2024.
+
+[package]
+name = "tockloader-lib"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-trait = "0.1.73"
+bytes = "1.5.0"
+clap = { version = "4.1.1", features = ["cargo"] }
+console = "0.15.7"
+enum_dispatch = "0.3.12"
+futures = "0.3.28"
+tokio = { version = "1.32.0", features = ["full"] }
+tokio-serial = {version = "5.4.4", features = ["libudev"]}
+tokio-util = { version = "0.7.8", features = ["full"] }
+tock-process-console = { path = "../tock-process-console/" }
+probe-rs = "0.24.0"
+tbf-parser = { path = "../tbf-parser"}
+glob = "0.3.1"
+inquire = "0.7.5"
+utf8-decode = "1.0.1"
+byteorder = "1.5.0"
+tar = "0.4.41"
+
```

Comment:
Are all of these dependencies needed?

Reply:
Removed tar; the other two are used by files from "attributes" folder.

Reply:
Oh, I meant to highlight the entirety of `Cargo.toml`. Could you also check if the other dependencies, besides these 3 higlighted here, are also still needed?